app/(api)/api/v1/test-error-handling/route.ts

```
@@ -0,0 +1,49 @@
+    case "validation":
+      return new ValidationError("Invalid data", {
+        field: "name",
+        message: "Name is required",
+      });
+    case "not-found":
+      return new NotFoundError("User", 123);
+    case "auth":
+      return new AuthenticationError();
+    case "database":
+      // Simulate a Prisma error structure without importing Prisma
+      const error = new Error("Database connection failed") as any;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)